src/fitCombo.jl

```diff
@@ -0,0 +1,72 @@
+""" In this file the data will be organized for combinations to be fit to the model. """
+
+########### 1. control, Palbociclib 50, Palbociclib 50 nM + lapatinibs [25 nM, 50 nM, 100 nM, 250 nM]:
+########### 2. control, Palbociclib 50, Palbociclib 50 nM + gemcitabines [5 nM, 10 nM, 17 nM, 30 nM]
+########### 3. control, Gemcitabine 10, Gemcitabine 10 nM + palbociclibs [25 nM, 50 nM, 100 nM, 250 nM]
+########### 4. control, Gemcitabine 10, Gemcitabine 10 nM + lapatinibs [25 nM, 50 nM, 100 nM, 250 nM]
+########### 5. control, Doxorubicin 20, Doxorubicin 20 nM + gemcitabines [5 nM, 10 nM, 17 nM, 30 nM]
+########### 6. control, Lapatinib  100, Lapatinib  100 nM + palbociclibs [25 nM, 50 nM, 100 nM, 250 nM]
+
+""" To fit combinations. """
+function RESID_through_bliss(p, g1c, g2c)
+    p_combo = my_helper(p)
+    t = LinRange(0.0, 0.5 * size(g1c, 1), size(g1c, 1))
+    res = 0
+    for j=1:5
+        for i=1:6
+            res += predict(p_combo[:, i, j], p_combo[:, 1, j], t, g1c[:, i, j], g2c[:, i, j])[1]
+        end
+    end
+    res
```

Comment:
Here is the residual function. We have 5 pairs of combinations in the data, and 6 concentrations [control, drugA, drugA+DrugB1, drugA+DrugB2, drugA+DrugB3, drugA+DrugB4]